skyeye-config-example.yaml

```diff
@@ -0,0 +1,10 @@
+log-level: info
```

Comment:
Let's leave default values out of the file for now, some values like the mandatory threat radius may change with testing and it's easy to forget to change this file.

Reply:
This file should just be an example on how to specify variables.  We don't load it